src/test/ui/infinite/infinite-tag-type-recursion.stderr

```diff
@@ -6,7 +6,14 @@ LL | enum MList { Cons(isize, MList), Nil }
    | |
    | recursive type has infinite size
    |
-   = help: insert indirection (e.g., a `Box`, `Rc`, or `&`) at some point to make `MList` representable
+help: insert some indirection to make `MList` representable
+   |
+LL | enum MList { Cons(isize, Box<MList>), Nil }
+   |                          ^^^^     ^
+LL | enum MList { Cons(isize, Rc<MList>), Nil }
+   |                          ^^^     ^
+LL | enum MList { Cons(isize, &MList), Nil }
+   |                          ^
```

Comment:
This suggestion isn't entirely correct, but if used the compiler will correctly tell you to write

```rust
enum MList<'a> { Cons(isize, &'a MList), Nil }
```

so I think it is ok.